### DISCUSSION TITLE
need plantuml in starlight

### DISCUSSION THREAD

**Discussion by moatkon** [withastro/starlight#1615] [Feature Requests]
### What version of `starlight` are you using?

0.19.1

### What is your idea?

need plantuml in starlight

### Why is this feature necessary?

plantuml, use code to generate ...

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by stereobooster** [comment]
related https://github.com/withastro/starlight/discussions/1259

**Comment by HiDeoo** [comment]
Sharing the [Starlight community tools and integrations page](https://starlight.astro.build/resources/plugins/#community-tools-and-integrations) here, as it contains [`astro-plantuml`](https://github.com/joesaby/astro-plantuml).

We currently don't have any plan to add such support natively in Starlight considering they can be easily implement as [Astro integrations](https://astro.build/integrations/) (which Starlight is built on top of) or [Starlight plugins](https://starlight.astro.build/resources/plugins/) and make maintaining the codebase easier.
